src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -97,8 +97,7 @@ int MPIDI_OFI_handle_cq_error(int vni, int nic, ssize_t ret);
         mpi_errno = MPIDI_OFI_retry_progress();                      \
         MPIDI_OFI_THREAD_CS_ENTER_VCI_OPTIONAL(vci_);			     \
         MPIR_ERR_CHECK(mpi_errno);                               \
-        _retry--;                                           \
-    } while (_ret == -FI_EAGAIN);                           \
+    } while (1);                                            \
```

Comment:
Did you mean to change this loop condition? Otherwise it looks good.

Reply:
Yes. All other errors (including MPI_SUCCESS) are checked and will break out of the loop already.